lib/form/html.js

```diff
@@ -29,7 +29,46 @@ const addError = (html, input, errors) => {
     });
   }
 
-  result += html;
+  result += render;
+
+  return result;
+};
+
+/*
+ * Add Required Rendered Inputs
+ *
+ * @param {string} html - Rendered HTML
+ * @param {InputType} type - Input type being rendered
+ *
+ * @returns {string} - Rendered HTML, with required indicators
+ */
+const addRequired = (html, input) => {
+  let render = html;
+  let result = '';
+
+  Object.keys(input.inputs).map(inp => {
+    if (((input.hasOwnProperty('required')) && input.required === 'save') || ((input.inputs[inp].hasOwnProperty('required')) && input.inputs[inp].required === 'save')) {
```

Comment:
As both of the are `&&` checks, don't need the `hasOwnProperty`. If you try and access the property and it doesn't exist, it'll be `undefined`, so it won't equal `save`. Other places we used `hasOwnProperty` were because you were checking to see if the property `=== undefined`, which wasn't what you wanted to check (because the property could exist _and_ be `undefined`)